annotationProcessor/src/main/java/org/corfudb/annotations/ObjectAnnotationProcessor.java

```diff
@@ -920,6 +922,83 @@ private void addUndoMap(TypeSpec.Builder typeSpecBuilder, TypeName originalName,
 
     }
 
+    private void addEntryToConflictMap(TypeSpec.Builder typeSpecBuilder, TypeName originalName,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "originalName". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)